Document list request by externalRef endpoint (#1108)
centrapay/centrapay-docs: src/content/api/payment-requests.mdx

---

<Endpoint
  path="/api/payment-requests/external-ref/{externalRef}"
  filename="payment-requests-list-by-external-ref"
>
  ## List Payment Requests by External Reference

>   This endpoint allows you to list [Payment Requests](#payment-request-model) that match the given external reference.
>   Results are [paginated](/api/pagination/).

Reviewer: ```suggestion
This endpoint returns a list of [Payment Requests](#payment-request-model) that match the given external reference.
Results are [paginated](/api/pagination/).
```
